cli/config/config.go

```diff
@@ -295,6 +401,11 @@ func (c *CommonOpts) GetParentMrn() string {
 }
 
 func (c *CommonOpts) UpstreamApiEndpoint() string {
+	// First check for environment variable override
+	if envEndpoint := os.Getenv("MONDOO_API_ENDPOINT"); envEndpoint != "" {
```

Comment:
The loading of environment variables like this one happens automatically, why do we need this?

Reply:
Left overs from when I was passing this stuff as flags in cnspec, removed.